src/components/single-line-diagram.js

```diff
@@ -183,7 +187,117 @@ const SingleLineDiagram = forwardRef((props, ref) => {
     };
 
     useLayoutEffect(() => {
+        function addNavigationArrowDef() {
+            let svg = document.getElementById('sld-svg').children[0]; //Get svg element
+            let defElement = document.createElementNS(
+                'http://www.w3.org/2000/svg',
+                'defs'
+            );
+
+            let html;
+            if (theme === LIGHT_THEME) {
```

Comment:
This is not a correct way to use theme. Everything should be contained in the theme to be able to add another one without changing the code. 
Why did you duplicate the html code in the 2 parts of the if? Why just not using a color from the theme to fill the jtml code?